previews/azure-functions-service-bus.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Azure Functions with Azure Service Bus
+component: ASBFunctions
+summary: Azure Functions Preview
+related:
+ - samples/previews/azure-functions
+reviewed: 2020-06-18
+---
+
+Host NServiceBus endpoints with [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/).
+
+## Basic Usage
+
+### Endpoint configuration
+
+snippet: endpoint-configuration
+
+The endpoint is automatically configured with the endpoint name and the transport connection string based on the values defined in the `[ServiceBusTrigger]` attribute using the `ServiceBusTriggeredEndpointConfiguration.FromAttributes` method.
+
+Alternatively, the endpoint name can be passed in manually:
+
+snippet: alternative-endpoint-setup
+
+### Azure Function definition
+
+snippet: function-definition
+
+## Configuration
+
+## License
+
+The license can be provided via the `NSERVICEBUS_LICENSE` environment variable which can be set via the Function settings in the Azure Portal.
+For local development, `local.settings.json` can be used. In Azure, specify a Function setting using environment variable as the key.
+
+include: license-file-local-setting-file
+
+### Custom diagnostics
+
+snippet: enable-diagnostics
```

Comment:
```suggestion
[NServiceBus startup diagnostics](nservicebus/hosting/startup-diagnostics.md) are disabled by default when using Azure Functions. Diagnostics can be written to the logs via the following snippet:

snippet: enable-diagnostics
```

Reply:
Done in https://github.com/Particular/docs.particular.net/commit/57781726dbfef65b0f43bccb926692d130b3321b along with ASB ASQ version.